pallets/collator-assignment/src/lib.rs

```diff
@@ -159,16 +169,110 @@ pub mod pallet {
     }
 
     impl<T: Config> Pallet<T> {
+        pub(crate) fn order_paras_with_core_config(
```

Comment:
Add docs here to explain what this function does. Especially for the returned boolean. You can expand on the edge cases if you think they are important.

```
/// Orders parachains and parathreads based on priority fee.
/// Returns a list of `ChainNumCollators` with the chains that were selected,
/// and a boolean indicating if `need_to_charge_tip`
```

Reply:
Now I realize that the order is not always by priority fee, so explain that please

Reply:
*fixed* (Please let me know if it is not clear, I can elaborate more.)

Reply:
Ok, the comments inside the function already explain the edge cases. Just add this or similar, to explain the boolean:

```
/// Returns a list of `ChainNumCollators` with the chains that were selected,
/// and a boolean indicating if `need_to_charge_tip`.
/// Will be true if chains were sorted by tip.
```

Am I correct to say that the boolean will be true if and only if the paras were sorted?

Reply:
*fixed*